packages/builder/src/registry.ts

```diff
@@ -27,25 +27,25 @@ export abstract class CannonRegistry {
   // that is a direct service resolve
   // ex @ipfs:Qm... is ipfs://Qm...
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  async getUrl(packageRef: string, chainId: number): Promise<string | null> {
+  async getUrl(serviceRef: string, chainId: number): Promise<string | null> {
     // Check if its an ipfs hash / url, if so we make sure to remove any incorrectly appended presets (like @main);
-    if (packageRef.startsWith('@')) {
-      const result = packageRef.replace(':', '://').replace('@', '');
-
-      return result.indexOf('@', 1) !== -1 ? result.slice(0, result.indexOf('@', 1)) : result;
+    if (serviceRef.startsWith('@')) {
+      const result = serviceRef.replace(':', '://').replace('@', '');
+      return result.indexOf('@') !== -1 ? result.slice(0, result.indexOf('@')) : result;
     }
 
     return null;
   }
 
+  // TODO: why is this empty?
```

Comment:
It's empty because there's never a case where we would want to return data in this superclass of the Canon registry whose purpose is to simply be a base common implementation

Perhaps we can make it abstract by having the function be a headless definition?